src/Components/Patient/PatientInfoCard.tsx

```diff
@@ -354,18 +378,18 @@ export default function PatientInfoCard(props: {
                         </div>
                       </div>
                     )}
-                  {(
-                    consultation?.consent_records?.filter((c) => !c.deleted) ||
-                    []
-                  ).length < 1 && (
-                    <div>
-                      <div className="inline-flex w-full items-center justify-start rounded border border-gray-500 bg-red-400 p-1 px-3 text-xs font-semibold leading-4">
-                        <span className="font-semibold text-white">
-                          Consent Records Missing
-                        </span>
+                  {!consentFilesLoading &&
+                    !consentRecordsLoading &&
```

Comment:
Can be rendered when consentRecords Load, and the Card can be further enriched with files ones that's available in the state